tablo/geom_utils.py

```diff
@@ -437,4 +438,51 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
```

Comment:
Makes sense. I move `esri_feature_to_ewkt` too; I think it can be part of the same collection.